### PR TITLE
Subscription examples

### PR DESCRIPTION
**Pull request by Yakimych**

### CODE REVIEW

File: example/src/TopCardsDisplayer.re [hunk 1]
```diff
@@ -30,6 +40,31 @@ module SiteStatisticsFragment = [%relay.fragment
 let make = (~siteStatistics as siteStatisticsRef) => {
   let siteStatistics = SiteStatisticsFragment.use(siteStatisticsRef);
 
+  let environment = ReasonRelay.useEnvironmentFromContext();
+  React.useEffect0(() => {
```
Comment (zth) [inline example/src/TopCardsDisplayer.re:44]: Will this run every render? I don't remember what 0 does 😅

File: example/src/TopCardsDisplayer.re [hunk 2]
```diff
@@ -30,6 +40,31 @@ module SiteStatisticsFragment = [%relay.fragment
...
+    let subscription =
+      CurrentVisitorsSubscription.subscribe(
+        ~environment,
+        ~variables=(),
+        ~onNext=
+          (response: CurrentVisitorsSubscription.Types.response) => {
```
Comment (zth) [inline example/src/TopCardsDisplayer.re:50]: Just curious - did inference not work properly here?

**Review by zth — APPROVED** [review]
This is really great! 😍 Thank you! Just 2 questions from me.

File: example/src/TopCardsDisplayer.re (hunk 1 above)
Comment (Yakimych) [reply example/src/TopCardsDisplayer.re:44]: Good point, had to check - we wouldn't want to subscribe on every render now, would we? :)
`useEffect` runs on every render. `useEffect0` runs once on mount and never re-runs. Same as `useEffect1` with an empty deps array, if I am not mistaken.

File: example/src/TopCardsDisplayer.re (hunk 2 above)
Comment (Yakimych) [reply example/src/TopCardsDisplayer.re]: Oh, it worked perfectly, sorry. Just my habit from getting burned by type-inference-related problems in F# that kicks in, so now I annotate most functions. Removed this for consistency with the rest of the codebase.

File: example/src/TopCardsDisplayer.re (hunk 1 above)
Comment (zth) [reply example/src/TopCardsDisplayer.re:44]: Excellent!

**Comment by zth** [comment]
Thank you for this! 🙏

**Pull request merged**